packages/framework/block-std/src/scope/block-std-scope.ts

```diff
@@ -47,6 +51,8 @@ export class BlockStdScope {
     this.spec = new SpecStore(this);
     this.view = new ViewStore(this);
     this.clipboard = new Clipboard(this);
+    this.container = new Container();
+    this.container.addFactory(BlockStdScope, () => this);
```

Comment:
Why using the internal `addFactory` instead of `addImpl` here? Not quite sure about what's the proper concept for the DI users to call.

Reply:
To keep the instance is the same.